### PR TITLE
add VMware-tanzu helm repository

### PR DESCRIPTION
**Pull request by cpanato**
Fixes: https://github.com/vmware-tanzu/helm-charts/issues/5

/cc @carlisia

### CODE REVIEW

**Comment by mattfarina** [comment]
@carlisia are you ok being listed as the maintainer on this charts repository?

File: repos.yaml [hunk 1]
```diff
@@ -729,6 +729,11 @@ repositories:
     maintainers:
       - email: tom.dickman@foldapp.com
         name: Tom Dickman
+  - name: vmware-tanzu
+    url: https://vmware-tanzu.github.io/helm-charts/
+    maintainers:
+      - email: carlisia@vmware.com
+        name: carlisia
```
Comment (carlisia) [inline repos.yaml]: Please also add:
```
- email: brubakern@vmware.com
  name: Nolan Brubaker
```

**Comment by carlisia** [comment]
Hey Matt, yes, thanks for checking. Adding @nbr would also be good.

File: repos.yaml (hunk 1 above)
Comment (cpanato) [reply repos.yaml]: adding

Comment (cpanato) [reply repos.yaml]: done

**Pull request merged**